client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -169,11 +169,11 @@ const EndDateField = props => {
 const productSaleModel = {
 	appliesToProductSale,
 	productAndSalePrice: {
-		labelText: translate( 'Product & Sale Price' ),
+		labelText: translate( 'Product & sale price' ),
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
 			salePrice: {
-				component: <CurrencyField labelText={ translate( 'Product Sale Price' ) } isRequired />,
+				component: <CurrencyField labelText={ translate( 'Product sale price' ) } isRequired />,
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Product & Sale Price' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update-sentence-case)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).